src/Hedgehog/CSharp/Property.fs

```diff
@@ -0,0 +1,136 @@
+namespace Hedgehog.CSharp
+
+#if !FABLE_COMPILER
+
+open System
+open System.Runtime.CompilerServices
+open Hedgehog
+
+[<Extension>]
+[<AbstractClass; Sealed>]
+type Property private () =
+
+    static member Failure : Property<unit> =
+        Property.failure
+
+    static member Discard : Property<unit> =
+        Property.discard
+
+    static member Success (value : 'T) : Property<'T> =
+        Property.Success(value)
```

Comment:
Shouldn't this be `success` (with lower `s`)?

Reply:
Yes, definitely needs to be a lowercase `s`.  Otherwise that method is calling itself.

I was thinking of suggesting that all these implementations be fully qualified.  Then this would have been easier to spot since
```fsharp
Hedgehog.Property.Success value
```
doesn't compile.

However, I was surprised to discover that
```fsharp
Hedgehog.Property.success value
```
doesn't compile either.  Even though Intellisense suggests completing `Hedgehog.Property.s` with the above, it doesn't compile because `Hedgehog.Property` resolves to the type `Property` instead of the module `Property`.